testgrid/config/config.yaml

```diff
@@ -4602,6 +4606,13 @@ dashboards:
     test_group_name: pull-federation-verify
     base_options: 'width=10'
 
+- name: istio
+  dashboard_tab:
+  - name: api-postsubmit
+    test_group_name: istio-api-postsubmit
+    alert_options:
+      alert_mail_to_addresses: 'some-email-address'
```

Comment:
This should be an actual email address, or we'll get spammed with errors in the logs on trying to send emails.

Reply:
yeah I'm asking @chxchx for an actual email address :-)

Reply:
Oh, that makes way more sense. XD Carry on!

Reply:
istio-oncall@googlegroups.com is the email we want to use.

Reply:
done